### PR TITLE
OCPBUGS-10568: migrate to using lease objects for leader election

### PR DESCRIPTION
**Pull request by tkashem**

### CODE REVIEW

File: test/e2e-encryption-perf/encryption_perf_test.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ const (
 func TestPerfEncryptionTypeAESCBC(tt *testing.T) {
 	ctx := context.TODO()
 	clientSet := getPerfClients(tt)
-	library.TestPerfEncryptionTypeAESCBC(tt, library.PerfScenario{
+	library.TestPerfEncryption(tt, library.PerfScenario{
```
Comment (dgrisonnet) [inline test/e2e-encryption-perf/encryption_perf_test.go:30]: you'll need to set the new EncryptionProvider field of PerfScenario  to `configv1.EncryptionTypeAESCBC`

**Comment by stlaz** [comment]
@tkashem please fix your commit signatures

**Comment by tkashem** [comment]
@stlaz ptal

**Comment by dgrisonnet** [comment]
Closing in favor of https://github.com/openshift/cluster-authentication-operator/pull/615.

/close

**Pull request closed without merging**